build.gradle.kts

```diff
@@ -0,0 +1,219 @@
+
+dependencies {
+  compile(localGroovy())
+  compile(gradleApi())
+  compile("com.google.cloud.tools:appengine-plugins-core:0.9.8")
+
+  testCompile("commons-io:commons-io:2.4")
+  testCompile("junit:junit:4.12")
+  testCompile("org.hamcrest:hamcrest-library:1.3")
+  testCompile("org.mockito:mockito-core:2.23.4")
+}
+
+
+tasks.wrapper {
+  gradleVersion = "6.9"
+}
+
+tasks.jar.configure {
+  manifest {
+    attributes(
+      mapOf(
+        "Implementation-Title" to  project.name,
+        "Implementation-Version" to version,
+        "Built-By" to System.getProperty("user.name"),
+        "Built-Date" to Date(),
+        "Built-JDK" to System.getProperty("java.version"),
+        "Built-Gradle" to gradle.gradleVersion
+      )
+    )
+  }
+}
+
+/* TESTING */
+tasks.test.configure {
+  testLogging {
+    showStandardStreams = true
+    exceptionFormat = FULL
+  }
+}
+
+sourceSets {
+  create("integTest") {
+    compileClasspath += main.get().output
+    runtimeClasspath += main.get().output
+  }
+}
+
+configurations {
+  named("integTestCompile").get().extendsFrom(testCompileClasspath.get())
+  named("integTestRuntime").get().extendsFrom(testRuntimeClasspath.get())
+}
+
+tasks.register("integTest", Test::class.java) {
+  testClassesDirs = sourceSets.getByName("integTest").output.classesDirs
+  classpath = sourceSets.getByName("integTest").runtimeClasspath
+  outputs.upToDateWhen { false }
+}
+/* TESTING */
+
+
+/* RELEASING */
+tasks.register("sourceJar", Jar::class.java) {
+  from(sourceSets.main.get().allJava)
+  classifier = "sources"
+}
+
+tasks.register("javadocJar", Jar::class.java){
+  dependsOn(tasks.javadoc)
+  from(tasks.javadoc.map { it.destinationDir!! })
+  classifier = "javadoc"
+}
+
+project.afterEvaluate {
+  tasks.register("writePom") {
+    val outputFile = file("$buildDir/pom/${project.name}-${project.version}.pom")
+    outputs.file(outputFile)
+
+    doLast {
+      maven {
+        pom {
+          project {
+            setProperty("name", "App Engine Gradle Plugin")
+            setProperty(
+              "description",
+              "This Gradle plugin provides tasks to build and deploy Google App Engine applications."
+            )
+            setProperty("url", "https://github.com/GoogleCloudPlatform/app-gradle-plugin")
+            setProperty("inceptionYear", "2016")
+
+            setProperty(
+              "scm", mapOf(
+                "url" to "https://github.com/GoogleCloudPlatform/app-gradle-plugin",
+                "connection" to "scm:https://github.com/GoogleCloudPlatform/app-gradle-plugin.git",
+                "developerConnection" to "scm:git://github.com/GoogleCloudPlatform/app-gradle-plugin.git"
+              )
+            )
+
+            invokeMethod("licenses", delegateClosureOf<groovy.lang.GroovyObject> {
+              invokeMethod("license", delegateClosureOf<groovy.lang.GroovyObject> {
+                setProperty("name", "The Apache Software License, Version 2.0")
```

Comment:
Did you mean `withGroovyBuilder` here?
https://docs.gradle.org/current/userguide/kotlin_dsl.html#the_kotlin_dsl_groovy_builder

It would be something like

```kotlin
withGroovyBuilder {
    "licenses" {
        "license" {
            "name"("The Apache Software License, Version 2.0")
            "url"("http://www.apache.org/licenses/LICENSE-2.0.txt")
            ...
        }
    }
}
```